Jasmine/Common/ViewModels/LevelSelectViewModelProtocol.swift

```diff
@@ -0,0 +1,31 @@
+protocol LevelSelectViewModelProtocol {
+    /// The defaults levels in the game.
+    var defaultLevels: [GameInfo] { get }
+    /// The custom levels in the game.
+    var customLevels: [GameInfo] { get }
```

Comment:
That's true lol I thought you will refresh it by yourself. Too much web dev liao